src/impl_x86__base_implementation.inl

```diff
@@ -667,6 +667,9 @@ X86Microarchitecture GetX86Microarchitecture(const X86Info* info) {
       case CPUID(0x12, 0x01):
         // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
         return AMD_K12;
+      case CPUID(0x12, 0x00):
+        // https://www.amd.com/system/files/TechDocs/44739_12h_Rev_Gd.pdf
```

Comment:
I checked other cpuid LLANO http://users.atw.hu/instlatx64/AuthenticAMD/AuthenticAMD0300F10_K12_Llano_CPUID2.txt:

```
CPUID 00000001: 00300F10-00020800-00802009-178BFBFF
```
Family 0x12, Model 0x01, thus you can move your cpuid to AMD_K12 or cpuid(0x12, 0x01) to AMD_LLANO

@gchatelet, what name should we leave?

Reply:
We already have `K12` so we should reuse it to make sure we don't break someone.
So in the definition let's keep K12 with a comment `AMD_K12, // K12 LLANO` and here let's move the `CPUID` switch case up to return `AMD_K12`.